test/functional/migrations/show-command/command.ts

```diff
@@ -0,0 +1,26 @@
+import "reflect-metadata";
+import {createTestingConnections, closeTestingConnections, reloadTestingDatabases} from "../../../utils/test-utils";
+import {Connection} from "../../../../src/connection/Connection";
+
+describe("migrations > show command", () => {
+    let connections: Connection[];
+    before(async () => connections = await createTestingConnections({
+        migrations: [__dirname + "/migration/*.js"],
+        enabledDrivers: ["postgres"],
```

Comment:
Could we enable more drivers?

Reply:
There are no driver specific changes. I think run this test only for one driver is ok.

Reply:
I've enabled them all for now, if it adds too much of a time hit to the tests, we can drop to fewer drivers

Reply:
I'll drop back down to just postgres then 👌